packages/fast-components-styles-msft/src/checkbox/index.ts

```diff
@@ -99,12 +97,11 @@ const styles: ComponentStyles<ICheckboxClassNameContract, IDesignSystem> = {
             return config.foregroundColor;
         },
         ...applyTypeRampConfig("t7"),
-        marginLeft: toPx(5),
-        marginTop: toPx(2)
+        marginLeft: toPx(5)
     },
     checkbox_disabled: {
+        cursor: "not-allowed",
```

Comment:
Are we sure we want this? This isn't the "normal" treatment of a disabled checkbox

Reply:
@nalogiudice Confirmed with @Falkicon that we'll move forward this currently and revisit the overall story later on.